e2e-tests/cypress/e2e/journeys/insurance/policy-and-exports/multiple-contract-policy/save-and-back.spec.js

```diff
@@ -0,0 +1,163 @@
+
+const multiplePolicyFieldId = FIELD_IDS.INSURANCE.POLICY_AND_EXPORTS.POLICY_TYPE;
+const multiplePolicyField = typeOfPolicyPage[multiplePolicyFieldId].multiple;
+
+const {
+  INSURANCE: {
+    ROOT: INSURANCE_ROOT,
+    ALL_SECTIONS,
+    POLICY_AND_EXPORTS: {
+      MULTIPLE_CONTRACT_POLICY,
+    },
+  },
+} = ROUTES;
+
+const {
+  INSURANCE: {
+    POLICY_AND_EXPORTS: {
+      CONTRACT_POLICY: {
+        MULTIPLE: { TOTAL_SALES_TO_BUYER },
+      },
+    },
+  },
+} = FIELD_IDS;
+
+context('Insurance - Policy and exports - Multiple contract policy page - Save and go back', () => {
+  let referenceNumber;
+
+  before(() => {
+    cy.visit(ROUTES.INSURANCE.START, {
+      auth: {
+        username: Cypress.config('basicAuthKey'),
+        password: Cypress.config('basicAuthSecret'),
+      },
+    });
+
+    cy.submitInsuranceEligibilityAndStartApplication();
+
+    taskList.prepareApplication.tasks.policyTypeAndExports.link().click();
+
+    multiplePolicyField.input().click();
+    submitButton().click();
+
+    getReferenceNumber().then((id) => {
+      referenceNumber = id;
+
+      const expected = `${Cypress.config('baseUrl')}${INSURANCE_ROOT}/${referenceNumber}${MULTIPLE_CONTRACT_POLICY}`;
+      cy.url().should('eq', expected);
+    });
+  });
+
+  beforeEach(() => {
+    Cypress.Cookies.preserveOnce('_csrf');
+    Cypress.Cookies.preserveOnce('connect.sid');
+  });
+
+  describe('when submitting an empty form via `save and go back` button', () => {
+    it(`should redirect to ${ALL_SECTIONS}`, () => {
+      saveAndBackButton().click();
+
+      const expected = `${Cypress.config('baseUrl')}${INSURANCE_ROOT}/${referenceNumber}${ALL_SECTIONS}`;
+
+      cy.url().should('eq', expected);
+    });
+
+    it('should retain the `type of policy and exports` task status as `in progress`', () => {
+      const task = taskList.prepareApplication.tasks.policyTypeAndExports;
+
+      task.status().invoke('text').then((text) => {
+        const expected = TASKS.STATUS.IN_PROGRESS;
+
+        expect(text.trim()).equal(expected);
+      });
+    });
+  });
+
+  describe('when entering an invalid total sales to buyer and submitting the form via `save and go back` button', () => {
+    const field = multipleContractPolicyPage[TOTAL_SALES_TO_BUYER];
+    const invalidValue = 'Not a number';
+
+    before(() => {
+      // go back to the page via the task list
+      taskList.prepareApplication.tasks.policyTypeAndExports.link().click();
+      submitButton().click();
+
+      field.input().type(invalidValue);
+    });
+
+    it(`should redirect to ${ALL_SECTIONS}`, () => {
+      saveAndBackButton().click();
+
+      const expected = `${Cypress.config('baseUrl')}${INSURANCE_ROOT}/${referenceNumber}${ALL_SECTIONS}`;
+
+      cy.url().should('eq', expected);
+    });
+
+    it('should retain the `type of policy and exports` task status as `in progress`', () => {
+      const task = taskList.prepareApplication.tasks.policyTypeAndExports;
```

Comment:
can this be declared at the top so it can be used multiple times?

Reply:
Excellent idea - i've updated all policy and export E2E tests to have a single decleration of this